media/js/firefox/new-v1.js

```diff
@@ -0,0 +1,43 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+;(function($) {
+    'use strict';
+
+    var $html = $(document.documentElement);
+    var isIELT9 = (site.platform === 'windows' && /MSIE\s[1-8]\./.test(navigator.userAgent));
+
+    var queryWideViewport;
+
+    var loadFonts = function(mq) {
+        if (mq.matches) {
+            var fntOpenSans = new FontFaceObserver('Open Sans', {
+                weight: 'normal'
+            });
+            var fntOpenSansBold = new FontFaceObserver('Open Sans', {
+                weight: 'bold'
+            });
+            var fntOpenSansLight = new FontFaceObserver('Open Sans Light');
+
+            Promise.all([fntOpenSans.check(), fntOpenSansLight.check()]).then(function() {
```

Comment:
According to [the docs](https://github.com/bramstein/fontfaceobserver#installation), the version of `fontfaceobserver.js` used includes a small promise polyfill, so should be good there.